src/Selenium2Library/keywords/_browsermanagement.py

```diff
@@ -331,6 +331,14 @@ def select_window(self, locator=None):
             pass
         finally:
             self._window_manager.select(self._current_browser(), locator)
+            
+    def get_log(self, log_type):
+        """Get the log for a given selenium log type
+
+        The `log_type` argument defines which logs to get. Possible values are:
+        `browser`, `driver`, `client` or `server`
+        """
+        return self._current_browser().get_log(log_type)
 
```

Comment:
There is no validation on the values of `log_type`. If we pass `BROWSER` we would have an error.